nym-wallet/src/components/Receive/ReceiveModal.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { AppContext } from 'src/context';
+import { Box, Stack, Typography } from '@mui/material';
+import QRCode from 'qrcode.react';
+import { SxProps } from '@mui/system';
```

Comment:
One small thing here - Move the `SxProps` import into the `@mui/material` imports

Reply:
My IDE autoimport got me xD